guide/07-working-with-bigdata/07-use_proximity.ipynb

```diff
@@ -47,7 +47,7 @@
     "      <td><a href=\"https://enterprise.arcgis.com/en/portal/latest/use/geoanalytics-trace-proximity-events.htm\"><p align=\"left\">trace_proximity_events</p></a></td>\n",
```

Comment:
<p>All the text in the <strong>Create Buffers </strong>section displays as bold, including the paragraph text and the code text within the cells. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
![image](https://user-images.githubusercontent.com/1399179/184011483-7db2852c-a492-48e4-86ff-58e156b70bf3.png)